source/exe/posix/platform_checks.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <fcntl.h>
+#include <sys/resource.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+
+#include "envoy/filesystem/filesystem.h"
+
+#include "common/common/assert.h"
+
+namespace Envoy {
+
+static void check_sysctl_long(std::string path, long expected, Filesystem::Instance& file_system) {
+  if (file_system.fileExists(path)) {
+    long current = std::stol(file_system.fileReadToEnd(path));
+
+    if (current < expected) {
+      std::cerr << fmt::format("{} is set to {} please consider increasing it.", current, path)
```

Comment:
indicate out how to increase it, to make the message actionable by an Envoy operator.

Also you should use `ENVOY_LOG_MISC(error, ...` here.